website/source/docs/auth/oci.html.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: "docs"
+page_title: "OCI - Vault Auth Plugin"
+sidebar_title: "OCI"
+sidebar_current: "docs-auth-oci"
+description: |-
+  The OCI Auth plugin for HashiCorp (HC) Vault enables authentication and authorization using OCI Identity credentials.
+---
+
+# Onboarding to OCI HashiCorp Vault Auth Plugin
+
+The OCI Auth plugin for HashiCorp (HC) Vault enables authentication and authorization using OCI Identity credentials. The open source OCI Auth plugin for HC Vault has been tested with version 0.11.4 of HC Vault and requires Go.
+
+## Role-Based Authorization
+
+The OCI Auth plugin authorizes using roles. A role is defined as a set of allowed [vault policies](https://www.vaultproject.io/docs/concepts/policies.html) for specific entities. When an entity such as a user or instance logs into HC Vault, it requests a role for that login session. The OCI Auth plugin checks whether the entity is allowed to use the role and which HC Vault policies are associated with that role. It then assigns the given vault policies to the HC Vault login request.
```

Comment:
Instead of `HC Vault` lets just use `Vault`

Reply:
Done